apps/server/src/shared/infra/identity-management/keycloak/service/keycloak-identity-management-oauth.service.spec.ts

```diff
@@ -142,38 +164,56 @@ describe('KeycloakIdentityManagementService', () => {
 	});
 
 	describe('isOauthConfigAvailable', () => {
+		const setup = (testKcConnection: boolean) => {
+			kcAdminServiceMock.testKcConnection.mockResolvedValueOnce(testKcConnection);
+		};
+
 		describe('when keycloak is available', () => {
 			it('should return true', async () => {
-				kcAdminServiceMock.testKcConnection.mockResolvedValueOnce(true);
+				setup(true);
```

Comment:
Als Punkt zur Diskussion, wahrscheinlich über diesen PR hinausgehend: Hier hat man ja nur eine Zeile Setup und es macht den Code wesentlich schwieriger zu lesen mit der extra Setup-Funktion (die jetzt auch noch einen Parameter braucht). Vielleicht könnte man diese Regel, dass jedes Szenario eine Setup-Funktion haben muss, doch aufweichen und sagen, wenn es nur einzwei Zeilen sind, dann schreibt man das Szenario doch direkt ins `it` rein...